### PR TITLE
Made a linux script, then added node, and npm as features.

### PR DESCRIPTION
**Pull request by ChrisForti**

### CODE REVIEW

File: linux-deploy.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# node should be installed
+
+if (which node)
+then 
+ echo "node is installed"
+else
+ echo "installing node"
+ apt install node
```
Comment (stephenlauck) [inline linux-deploy.sh:8]: This is good. If you want a more up to date version you can try this too:

https://github.com/nodesource/distributions#deb

**Review by stephenlauck — APPROVED** [review]

File: linux-deploy.sh (hunk 1 above)
Comment (ChrisForti) [reply linux-deploy.sh:8]: Do you mean curl this link as a command?

Comment (stephenlauck) [reply linux-deploy.sh:8]: you'd put this in your script inside the "installing node" else statement:

```
curl -SLO https://deb.nodesource.com/nsolid_setup_deb.sh
chmod 500 nsolid_setup_deb.sh
./nsolid_setup_deb.sh 21
apt-get install nodejs npm -y
```

I also added npm on the install line just to consolidate them. I think when you install node this way you auto get npm but I put it on the install line.

Comment (ChrisForti) [reply linux-deploy.sh:8]: You know what I seen that minus the npm install, and thought it to be too verbose. For what I was trying to do, I'll change it when I get by my machine.

**Pull request merged**